tsl/test/sql/continuous_aggs_union_view.sql

```diff
@@ -0,0 +1,119 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+
+-- disable background workers to make results reproducible
+\c :TEST_DBNAME :ROLE_SUPERUSER
+SELECT _timescaledb_internal.stop_background_workers();
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+
+CREATE TABLE metrics(f1 int, f2 int, time timestamptz NOT NULL, device_id int, value float);
+SELECT create_hypertable('metrics','time');
+ALTER TABLE metrics DROP COLUMN f1;
+
+INSERT INTO metrics(time, device_id, value) SELECT '2000-01-01'::timestamptz, 1, 0.5;
+INSERT INTO metrics(time, device_id, value) SELECT '2000-01-01'::timestamptz, 2, 0.5;
+INSERT INTO metrics(time, device_id, value) SELECT '2000-01-01'::timestamptz, 3, 0.5;
+
+--
+-- test switching continuous agg view between different modes
+--
+
+-- check default view for new continuous aggregate
+CREATE VIEW metrics_summary WITH (timescaledb.continuous) AS SELECT time_bucket('1d',time), avg(value) FROM metrics GROUP BY 1;
+ALTER TABLE metrics DROP COLUMN f2;
+
+-- this should be union view
```

Comment:
```suggestion
-- this should be JIT view
```

I suggest also adding a line or two about what one should expect in the output to identify this as a JIT view.

Reply:
the expected output is a union view, there is no JIT view